plugins/WebsiteMeasurable/MeasurableSettings.php

```diff
@@ -387,9 +387,9 @@ private function makeSiteSearchKeywords()
         });
     }
 
-    private function makeSiteSearchCategory(Plugin\Manager $pluginManager)
+    private function makeSiteSearchCategory()
     {
-        return $this->makeProperty('sitesearch_category_parameters', $default = [], FieldConfig::TYPE_ARRAY, function (FieldConfig $field) use ($pluginManager) {
+        return $this->makeProperty('sitesearch_category_parameters', $default = [], FieldConfig::TYPE_ARRAY, function (FieldConfig $field) {
```

Comment:
`$pluginManager` isn't used anywhere in the anonymous function, so we do not need to pass it through all the way.